tests/simple.sh

```diff
@@ -0,0 +1,87 @@
+#/bin/bash
+# A quick port of the travis tests to bash
+# TODO evaluate using a framework..
+# We need to be running as root..
+
+PYTHON=${1:-/usr/bin/python3}
+SOSPATH=${2:-./sosreport}
+
+NUMOFFAILURES=0
+
+run_expecting_sucess () {
+FAIL=false
+#Make sure clean
+rm stderr stdout
+
+echo "######### RUNNING $1 #########"
+$PYTHON $SOSPATH $1 2> stderr 1> stdout
+
+if [ $? -eq 0 ]; then
+  echo "### Success"
+else
+  echo "!!! FAILED !!!"
+  FAIL=true
+fi
+
+if [ -s stderr ]; then
+   FAIL=true
+   echo "!!! FAILED !!!"
+   echo "### start stderr"
+   cat stderr
+   echo "### end stderr"
+fi
+
+echo "### start stdout"
+cat stdout
+echo "### end stdout"
+
+echo "######### DONE WITH $1 #########"
+
+if $FAIL; then 
+  return 1
+else
+  return 0
+fi
+
+}
+
+# If /etc/sos.conf doesn't exist let's just make it..
+if [ -f /etc/sos.conf ]; then
+   echo "/etc/sos.conf already exists"
+else
+   echo "Creating /etc/sos.conf"
+   touch /etc/sos.conf
+fi
+
+run_expecting_sucess " -l"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --list-presets"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --list-profiles"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --batch"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --batch -a --case-id 1010101 -c never --label TESTLABEL"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --batch --debug --experimental --quiet -z gzip"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --batch --no-report --no-env-vars --build --verbose -z bzip2"
+NUMOFFAILURES=$(($NUMOFFAILURES + $?))
+run_expecting_sucess " --batch -z xz -t 1 --log-size 1 -n lxd" #This fails with just lxd, why?
```

Comment:
This fails on Red Hat distro, as `lxd` plugin is not a `RedHatPlugin` (cf. https://github.com/sosreport/sos/blob/master/sos/plugins/lxd.py#L14). Could you use some more generic plugin?

Reply:
I've opened a bug to track it #1921 .. but I did add a workaround for Travis. For whatever reason travis, the lxd plugin and --log-size don't get along.